superset/connectors/base/models.py

```diff
@@ -27,6 +27,23 @@
 from superset.models.slice import Slice
 from superset.utils import core as utils
 
+METRIC_FORM_DATA_PARAMS = [
+    "metric",
+    "timeseries_limit_metric",
+    "size",
+    "x",
+    "y",
+    "metric_2",
+]
```

Comment:
@etr2460 [this](https://github.com/apache/incubator-superset/blob/master/CONTRIBUTING.md#chart-parameters) could be a good resource for the relevant params.

Reply:
note that `percent_metrics` is referenced in the function below because it's an array of metrics and not an individual one. It looks like I missed secondary_metric though. I'll refactor so that those are included in a constant as well.

Reply:
You're right, my bad!